internal/criocli/criocli.go

```diff
@@ -569,6 +572,13 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			EnvVars:   []string{"CONTAINER_RUNROOT"},
 			TakesFile: true,
 		},
+		&cli.StringFlag{
+			Name:      "imagestore",
+			Usage:     "Store newly pulled images in the specified path (imagestore), rather than the path provided by --root.",
```

Comment:
I'm fine with this wording, but I'd double check the length with @haircommander or @saschagrunert .    For this quick hit, maybe just:  "Store newly pulled images in the specified path."  Peter/Sascha?

Reply:
Yes. I think we can at least omit the `(imagestore)`.

Reply:
@saschagrunert Done.